include/superior_mysqlpp/prepared_statements/prepared_statement_base.hpp

```diff
@@ -180,15 +180,6 @@ namespace SuperiorMySqlpp
              */
             std::vector<detail::NullableBase*> nullableBindings;
 
-        public:
-            /**
-             * Appears unused, probably can be removed.
-             * Is defined exactly like (private) DBDriver::size_t, so
-             * it was presumably added for similiar purpose.
-             * !!!!!
-             */
-            using size_t = unsigned long long;
```

Comment:
It seems, size_t was was public type definition? So it's api breaking change?

Reply:
Of course.

Reply:
Please mention this in changelog